### PR TITLE
Update version to 0.8.0

### PR DESCRIPTION
**Pull request by 95-martin-orion**
Do not merge until after #299.

The primary motivation for this release is to avoid incompatibility with Cirq due to #278; however, enough features have been submitted since 0.7.1 that a new minor version is justified.

### CODE REVIEW

**Comment by MichaelBroughton** [comment]
Would this be able to hold off until a fix for #288 is merged in as well ? We need that feature to progress with noise in TFQ and @sergeisakov mentioned it would be coming before this week is out.

**Comment by 95-martin-orion** [comment]
> Would this be able to hold off until a fix for #288 is merged in as well?

The upcoming Cirq release (scheduled for end of this week) is incompatible with the latest qsimcirq release due to a disagreement in `_resolve_parameters_`. Without this release, `import cirq, qsimcirq` will misbehave for parameterized circuits.

As such, this release is relatively urgent. I'm fine with cutting another release for the #288 fix, even if it's shortly afterwards - qsim releases are pretty cheap.

**Comment by MichaelBroughton** [comment]
Sounds good. Interestingly enough we got burned by that exact same change.

**Review by MichaelBroughton — APPROVED** [review]

**Pull request merged**
